Breeding-Insight/bi-api
# [BI-2155] Cannot create a new list referencing existing GIDs

# Description
**Story:** [BI-2155](https://breedinginsight.atlassian.net/jira/software/c/projects/BI/boards/1?selectedIssue=BI-2155)

[BI-2017](https://github.com/Breeding-Insight/bi-api/pull/341) inadvertently dropped the germplasmAccessionNumber field from the json string stored in the database for the germplasm import template when the germplasmName field was updated. 

A migration was created to add the germplasmAccessionNumber field to the template map.


# Dependencies
none

# Testing

1. Import germplasm 
2. Import a new germplasm list that references the GIDs from step 1, but with different names, sources, or methods
3. Confirm that the import preview shows the names, sources, and methods for the GIDs from step 1, not from step 2.
4. Complete the import from step 2 and confirm in the germplasm list details table that only germplasm with GIDs from step 1 are present.


# Checklist:



[BI-2155]: https://breedinginsight.atlassian.net/browse/BI-2155?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[BI-2017]: https://breedinginsight.atlassian.net/browse/BI-2017?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
